src/java/org/apache/cassandra/auth/MutualTlsCertificateValidator.java

```diff
@@ -35,29 +35,55 @@ public interface MutualTlsCertificateValidator
      * Perform any checks that are to be performed on the certificate before making authorization check to grant the
      * access to the client during mTLS connection.
      *
-     * For example
-     *  - Verifying CA information
-     *  - Checking CN information
-     *  - Validating Issuer information
-     *  - Checking organization information etc
+     * <p>For example:
+     * <ul>
+     *  <li>Verifying CA information
+     *  <li>Checking CN information
+     *  <li>Validating Issuer information
+     *  <li>Checking organization information etc
+     * </ul>
      *
      * @param clientCertificateChain client certificate chain
-     * @return returns if the certificate is valid or not
+     * @return returns {@code true} if the certificate is valid, {@code false} otherwise
      */
-    boolean isValidCertificate(Certificate[] clientCertificateChain);
+    default boolean isValidCertificate(Certificate[] clientCertificateChain)
```

Comment:
worth marking deprecated?  From what I can tell its not used anymore.

Reply:
(this may be moot if you end up moving out the age verification from the validator)